### PR TITLE
Added new actions, removed deprecated actions, refactored code.

### PR DESCRIPTION
**Pull request by fr-rpathade**
#### What's New

 - Added following new actions and their playbooks:
    - Get Vulnerability Intelligence CVEs 
    - Get Vulnerability Intelligence CVEs By ID
    - Get Vulnerability Intelligence CVE exposures(hits) by CVE ID
    - Get Vulnerability Intelligence Statistics for CVE ID
    - Get Vulnerability Intelligence vulnerable Products
    - Get Vulnerability Intelligence vulnerable Vendors
    - Get Stealers Infections(Marketplaces) on Sale Count
    - Get Stealers Infections(Marketplaces) on Sale
    - Get Vendor Watch List
    - Get Vendor details for the given vendor ID
    - Get Vendor exposure for the given vendor ID
    - Get Ransomware Intelligence potential victims
    - Get Ransomware Intelligence victims
    - Get Ransomware Intelligence victim details by ID
    - Get Ransomware Intelligence vendors watch list (monitored/matched)
    - Get Ransomware Intelligence Org watch list (monitored/matched)
    - Get Ransomware Intelligence threat campaigns
    - Get Ransomware Intelligence Statistics
    - Get Ransomware Intelligence technical indicators for given ransomware group
    - Get Ransomware Intelligence ransomware group information

### What's Improved
- Re-factored connector code to reduce to feature specific file.

### What's remove

- Removed deprecated actions and their playbooks:
   - Vulnerability Intelligence deprecated actions
   - Removed deprecated actions for vendor monitoring
   - Removed deprecated actions for Ransomware Intelligence

#### Fix:
- Re-factored connector code to reduce to feature specific file.

### CODE REVIEW

File: fortinet-fortirecon-aci/info.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
   "name": "fortinet-fortirecon-aci",
   "label": "Fortinet FortiRecon ACI",
-  "version": "1.2.0",
+  "version": "2.0.0",
   "description": "FortiRecon is a Digital Risk Protection Service (DRPS) product that provides an outside-the-network view to the risks posed to your enterprise.The Adversary Centric Intelligence (ACI) module leverages FortiGuard Threat Analysts to provide comprehensive coverage of dark web, open source, and technical threat intelligence, including threat actor insights. This information enables administrators to proactively assess risks, respond faster to incidents, better understand their attackers, and protect assets. This connector facilitates the automated operations related to ACI. <br/><br/>This connector has a dependency on the <a href=\"/content-hub/all-content/?contentType=solutionpack&amp;tag=ThreatIntelManagement\" target=\"_blank\" rel=\"noopener\">Threat Intel Management Solution Pack</a>. Install the Solution Pack before enabling ingestion of Threat Feeds from this source.",
-  "publisher": "Fortinet",
+  "publisher": "Fortinet FortiRecon",
```
Comment (cs-utkarsh-surana) [inline fortinet-fortirecon-aci/info.json]: Change this to fortinet

File: fortinet-fortirecon-aci/info.json [hunk 2]
```diff
@@ -1795,178 +1364,84 @@
...
           "visible": true,
           "required": false,
-          "tooltip": "Select the addition type based on which you want to filter the result. You can select between EASM and MANUAL.",
-          "description": "Select the addition type based on which you want to filter the result. You can select between EASM and MANUAL.",
+          "tooltip": "Specify the approval status of the vendor, you can select between pending, approved, rejected.",
+          "description": "Specify the status of the vendor, you can select between pending, approved, rejected.",
           "options": [
-            "EASM",
-            "MANUAL"
+            "pending",
```
Comment (cs-utkarsh-surana) [inline fortinet-fortirecon-aci/info.json]: Capitalize the first letters of every option (e.g. Pending) do this which every parameters

File: fortinet-fortirecon-aci/info.json [hunk 3]
```diff
@@ -2277,95 +1717,1497 @@
...
+          "name": "sort",
+          "title": "Sort By Product Count",
+          "type": "select",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Select the sort by type based on which you want to sort the result. You can select between count:desc and count:asc.",
+          "description": "Select the sort by type based on which you want to sort the result. You can select between count:desc and count:asc.",
+          "options": [
+            "count:desc",
```
Comment (cs-utkarsh-surana) [inline fortinet-fortirecon-aci/info.json]: Can we change the option values to Accending and Decending

File: fortinet-fortirecon-aci/info.json [hunk 4]
```diff
@@ -2277,95 +1717,1497 @@
...
+          "name": "time_range_type",
+          "title": "Time Range Type",
+          "type": "select",
+          "editable": true,
+          "visible": true,
+          "required": false,
+          "tooltip": "Specify the range type to get the ransomware intelligence stats for the time range, you can select between all, last90days, last30days, last15days, last7days, current_month, previous_month.",
+          "description": "Specify the range type to get the ransomware intelligence stats for the time range, you can select between all, last90days, last30days, last15days, last7days, current_month, previous_month.",
+          "options": [
+            "all",
```
Comment (cs-utkarsh-surana) [inline fortinet-fortirecon-aci/info.json]: Values changes are required here as well.

**Review by cs-utkarsh-surana — CHANGES_REQUESTED** [review]

File: fortinet-fortirecon-aci/release_notes.md [hunk 5]
```diff
@@ -1,15 +1,33 @@
...
+    - Get Ransomware Intelligence victim details by ID
+    - Get Ransomware Intelligence vendors watch list (monitored/matched)
+    - Get Ransomware Intelligence Org watch list (monitored/matched)
+    - Get Ransomware Intelligence threat campaigns
+    - Get Ransomware Intelligence Statistics
+    - Get Ransomware Intelligence technical indicators for given ransomware group
+    - Get Ransomware Intelligence ransomware group information
+
+### What's Improved
+- Re-factored connector code
```
Comment (cs-utkarsh-surana) [inline fortinet-fortirecon-aci/release_notes.md]: Remove this

File: fortinet-fortirecon-aci/info.json (hunk 1 above)
Comment (fr-rpathade) [reply fortinet-fortirecon-aci/info.json]: Done

File: fortinet-fortirecon-aci/release_notes.md (hunk 5 above)
Comment (fr-rpathade) [reply fortinet-fortirecon-aci/release_notes.md]: Done

File: fortinet-fortirecon-aci/info.json (hunk 2 above)
Comment (fr-rpathade) [reply fortinet-fortirecon-aci/info.json]: Done

File: fortinet-fortirecon-aci/info.json (hunk 3 above)
Comment (fr-rpathade) [reply fortinet-fortirecon-aci/info.json]: Done

File: fortinet-fortirecon-aci/info.json (hunk 4 above)
Comment (fr-rpathade) [reply fortinet-fortirecon-aci/info.json]: Done

**Pull request merged**